packages/builders/src/components/selectMenu/SelectMenuOption.ts

```diff
@@ -8,23 +8,61 @@ import {
 } from '../Assertions';
 
 /**
- * Represents a validated option within a select menu component
+ * Represents a option within a select menu component
  */
-export class SelectMenuOptionBuilder extends UnsafeSelectMenuOptionBuilder {
-	public override setDescription(description: string) {
-		return super.setDescription(labelValueValidator.parse(description));
+export class SelectMenuOptionBuilder {
+	public constructor(public data: Partial<APISelectMenuOption> = {}) {}
+
+	/**
+	 * Sets the label of this option
+	 * @param label The label to show on this option
+	 */
+	public setLabel(label: string) {
+		this.data.label = label;
+		return this;
+	}
+
+	/**
+	 * Sets the value of this option
+	 * @param value The value of this option
+	 */
+	public setValue(value: string) {
+		this.data.value = value;
+		return this;
+	}
+
+	/**
+	 * Sets the description of this option.
+	 * @param description The description of this option
+	 */
+	public setDescription(description: string) {
+		this.data.description = labelValueValidator.parse(description);
```

Comment:
Thats not the right validator for this 👀

Reply:
It was like this before. Ig it's the same for both so I'll just rename it 
https://github.com/discordjs/discord.js/blob/358c3f4a46638fc818e65fcc113d84818bb68c10/packages/builders/src/components/selectMenu/SelectMenuOption.ts#L15